### ISSUE TITLE
VITS Finetuning question

### ISSUE THREAD

**Issue by seastar105** [espnet/espnet#4023]
In jvs/tts1/README.md, it says `Let us replace the tokens.txt and feats_stats.npz with pretrained model's one.` when finetuning VITS.

But, there's no feats_stats.npz when download pretrained vits model from zenodo. (https://zenodo.org/record/5414980#.YfsI_fVBxz8)

is it some typo thing? if not, how can i replace feats_stats.npz for finetuning as ar finetinung steps.

**Comment by kan-bayashi** [comment]
Sorry, that is typo.
You do not need to replace feats_stats.npz since vits did not use global mean normalization.
You can replace just onl tokens.txt.
I will fix the documents.
